lib/__testfixtures__/get-set.output.js

```diff
@@ -0,0 +1,8 @@
+import URL, { pathname_get as URL_pathname_get } from "std:global/URL";
+import { length_get as Array_length_get } from "std:global/Array";
+import { apply as Reflect_apply } from "std:global/Reflect";
+const arr = [1, 2, 3];
+arr.length = 5;
+const b = Reflect_apply(Array_length_get, arr);
```

Comment:
This should not be rewritten.

Reply:
So for data properties I shouldn't be rewriting getters or setters? I thought it was only setters previously

Reply:
No, it's both. For data properties there are no originals.